src/dmd/iasmdmd.d

```diff
@@ -2452,7 +2342,7 @@ void asm_make_modrm_byte(
         ubyte *puchOpcode, uint *pusIdx,
         code *pc,
         uint usFlags,
-        OPND *popnd, OPND *popnd2)
+        scope OPND[] opnds)
```

Comment:
the length is always 2, you could do `scope ref OPND[2] opnds`

Reply:
The length is 1 or 2.  In cases where null was pass previously, not a single element slice is passed.

Reply:
OK.